src/components/Footer/Footer.tsx

```diff
@@ -1,6 +1,47 @@
 import React from 'react';
-import s from './Footer.module.scss';
+import { NavLink } from 'react-router-dom';
+import { Logo } from '../Logo/Logo';
+import '../../styles/blocks/_container.scss';
+import './Footer.scss';
 
-export const Footer: React.FC = () => {
-  return <div className={s.footer}></div>;
+const footerLinks = [
+  {
+    title: 'Github',
+    path: 'https://fe-may23-BugHunters.github.io/product_catalog_fe',
+  },
+  { title: 'Contacts', path: '/' },
+  { title: 'Rights', path: '/' },
+];
+
+export const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="container">
+        <div className="footer__content">
+          <div className="footer__logo">
+            <Logo />
+          </div>
+
+          <nav className="footer__nav nav">
+            <ul className="nav__list">
+              {footerLinks.map((link) => (
+                <li key={link.title} className="nav__item">
+                  <NavLink to={link.path} target="_blank" className="nav__link">
+                    {link.title}
+                  </NavLink>
+                </li>
+              ))}
+            </ul>
+          </nav>
+
+          <div className="footer__moveUp">
+            <span className="footer__label">Back to top</span>
+            <a href="#root" className="footer__btn-wrapper">
+              <div className="footer__btn" />
+            </a>
```

Comment:
#root triggers router and show not found
so, we have to use your previous function and make this button with <button>